src/main/java/au/gov/geoscience/portal/server/services/filters/PetroleumTenementFilter.java

```diff
@@ -70,15 +69,24 @@ public String getFilterStringAllRecords() {
     }
 
     public String getFilterStringBoundingBox(FilterBoundingBox bbox) {
-        List<String> localFragment = new ArrayList<String>(fragments);
+        List<String> localFragment = new ArrayList<>(fragments);
         if (bbox != null) {
             localFragment.add(this.generateBboxFragment(bbox, "pt:shape"));
         }
         return this.generateFilter(this.generateAndComparisonFragment(localFragment.toArray(new String[localFragment.size()])));
     }
 
     public String getFilterWithAdditionalStyle() {
-        List<String> localFragment = new ArrayList<String>(fragments);
+        List<String> localFragment = new ArrayList<>(fragments);
         return this.generateFilter(this.generateAndComparisonFragment(localFragment.toArray(new String[localFragment.size()])));
     }
+
+    // The Northern Territory services returns owner/holder names with a \n in front that causes the filter to fail
+    public String generatePropertyIsLikeFragmentWithStringReplace(String propertyName, String value) {
```

Comment:
Function could more accurately be named something along the lines of 
generatePropertyIsLikeFragmentExcludingNewlineCharacters()
Or something along those lines, because the main feature of the function compared to a plain filter is the removal of the newlines, and the string replace aspect isn't configurable for different replacements.